sockeye/inference.py

```diff
@@ -159,9 +160,10 @@ def sym_gen(source_seq_len: int):
             source_length = utils.compute_lengths(source)
 
             # source embedding
+            source_factors = [mx.sym.Variable(C.SOURCE_NAME + "_factor_" + str(i)) for i in range(self.config.num_factors)]
```

Comment:
[nit]: I would move variable declaration up to line 159, next to the source variable.

Reply:
done